src/ducks/stats/services.js

```diff
@@ -0,0 +1,33 @@
+import {
+  startOfMonth,
+  endOfMonth,
+  subMonths,
+  differenceInMonths
+} from 'date-fns'
+import { BankTransaction } from 'cozy-doctypes'
+import { sumBy } from 'lodash'
+
+export const getPeriod = () => {
+  const today = new Date()
+  const end = endOfMonth(subMonths(today, 1))
```

Comment:
This will give you a locale-based end of month. For a server in UTC+2, you will get a time 23:59:59 UTC+2, so 22:00:00Z. Are you sure this is want you want ?

Reply:
I refactored it using moment so we manipulate UTC dates.